src/chain_spec.rs

```diff
@@ -487,6 +497,27 @@ macro_rules! protocol_fee {
     };
 }
 
+macro_rules! rewards {
+    () => {
+        pallet_rewards::GenesisConfig {
+            itn_rewards: itn_rewards(),
+        }
+    };
+}
+
+#[allow(unreachable_code)]
+fn itn_rewards() -> Vec<(AccountId, Balance)> {
+    #[cfg(feature = "runtime-benchmarks")]
+    return Vec::new();
+
+    let itn_rewards_file = include_str!("data/itn_rewards.json");
+    serde_json::from_str::<Vec<(HexAccountId, Balance)>>(&itn_rewards_file)
+        .unwrap()
+        .into_iter()
+        .map(|(acc, bal)| (acc.0.into(), bal))
+        .collect()
+}
```

Comment:
```suggestion
fn itn_rewards() -> Vec<(AccountId, Balance)> {
    if cfg!(feature = "runtime-benchmarks") {
        return Vec::new();
    } else {
        let itn_rewards_file = include_str!("data/itn_rewards.json");
        serde_json::from_str::<Vec<(HexAccountId, Balance)>>(&itn_rewards_file)
            .unwrap()
            .into_iter()
            .map(|(acc, bal)| (acc.0.into(), bal))
            .collect()
    }
}
```

Reply:
I used prior art here. We use almost the exact same code for classic tickers. If we think this should be changed, classic tickers should be changed as well. We should probalbly be even more precise than what is suggested above.